### PR TITLE
Edited java servlet file (Hello World)

### PR DESCRIPTION
**Pull request by Erik-Handeland**

### CODE REVIEW

**Comment by arunk054** [comment]
Couple of points - 1) You need to add me as a reviewer so I know when the PR is ready to take a look. 2) Try to provide a description of what this PR is about, any test cases, etc.

File: portfolio/src/main/webapp/js/script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+function getMessages() {
+    fetch('/data').then(response => response.text()).then((message) => {
+    var data = eval(message); 
+
+    for (i in data)
+    {
+        var li = document.createElement("li");
+        li.innerHTML = "<span id='username' >" + data[i].userName+ "</span>" + "<span id='date'>   &nbsp;" + data[i].postDate + "</span> <br>" + "<span id='message'>" + data[i].userMessage + "</span>";
+        document.getElementById("comments").appendChild(li);
```
Comment (arunk054) [inline portfolio/src/main/webapp/js/script.js:23]: It is recommended to split such large string concatenations into smaller chunks. Even if you decide to put them all in one line, split the lines so each one is <= 80 chars.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +15,77 @@
...
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    //gets user data from html form
+     String userName = request.getParameter("username"); //"username" is html input name;
+     String message = request.getParameter("message");
+    Date timestamp = new Date();
+
+    Entity commentEntity = new Entity("Comments"); //creates entitiy that stores properties similar to a data structure
```
Comment (arunk054) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:76]: for reasonably large comments you can put them in a separate line before the target line of code.

File: portfolio/src/main/webapp/js/script.js (hunk 1 above)
Comment (Erik-Handeland) [reply portfolio/src/main/webapp/js/script.js:23]: Thanks, I'll fix the issue on my next push

**Pull request merged**